packages/react-magma-dom/src/components/NativeSelect/NativeSelect.test.js

```diff
@@ -108,4 +103,35 @@ describe('NativeSelect', () => {
 
     expect(getByText(errorMessage)).toBeInTheDocument();
   });
+
+  describe('additional content', () => {
+    const helpLinkLabel = 'Learn more';
+
+    const onHelpLinkClick = () => {
+      alert('Help link clicked!');
+    };
+
+    it('Should accept additional content to the right of the native select label', () => {
```

Comment:
This just tests it's in the DOM, do we want to add tests for the multiple scenarios? top vs inline block, etc

Reply:
Pfft yeah we do, added a test.